### PR TITLE
Refactor/examples

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Fixes a lot of stuff around the examples. Not yet final at all but I would like to bring the progress over to our future branch as well as the origin therefore I thought I could make an intermediate PR.

### CODE REVIEW

File: examples/static/samples/files/Product_Browse_Page.html [hunk 1]
```diff
@@ -0,0 +1,517 @@
+<!---
```
Comment (sebastianbenz) [inline examples/static/samples/files/Product_Browse_Page.html:1]: Why is this file being added?

File: examples/static/samples/files/amp-iframe.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
+<!--
```
Comment (sebastianbenz) [inline examples/static/samples/files/amp-iframe.html:1]: why is this file being added?

File: examples/static/samples/json/examples.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+ "items": [
+   {
+     "title": "amp-carousel",
+     "url": "/components/amp-carousel/"
```
Comment (sebastianbenz) [inline examples/static/samples/json/examples.json:5]: these URLs will also need to be fixed

File: examples/static/samples/json/related_articles.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+ "items": [
+   {
+     "title": "Shopping websites with AMP",
+     "thumbnail": "/static/samples/img/shopping_thumb.jpg",
+     "url": "/samples_templates/product/preview"
```
Comment (sebastianbenz) [inline examples/static/samples/json/related_articles.json:6]: these URLs need to be fixed

File: pages/content/amp-dev/documentation/components/reference/amp-accordion.md [hunk 5]
```diff
@@ -1,17 +1,15 @@
...
-    immediate children is considered a section in the accordion. Each of these
-    nodes must be a ` tag.
 $category@: layout
 formats:
   - websites
   - email
   - ads
+teaser:
+  text: >-
+    Provides a way for viewers to have a glance at the outline of the content
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/components/reference/amp-accordion.md:9]: The teaser is also contained in the JSON object you get from the ABE API.

**Review by sebastianbenz — COMMENTED** [review]
TBH, I'm not a fan of the `/static/samples/` prefix for images:

1. makes markup more verbose
2. needs to be documented for sample authors somewhere

File: pages/content/amp-dev/documentation/components/reference/amp-accordion.md (hunk 5 above)
Comment (matthiasrohmer) [reply pages/content/amp-dev/documentation/components/reference/amp-accordion.md:9]: Sorry, polluted PR. That's a part of the reference documentation, not ABE.

**Comment by matthiasrohmer** [comment]
> TBH, I'm not a fan of the `/static/samples/` prefix for images:
> 
> 1. makes markup more verbose
> 2. needs to be documented for sample authors somewhere

Yes I remembered that. But since neither the inherited static files from the docs nor the samples' statics are especially tidy I thought we should at least scope them to avoid conflicts. I totally get your point that this could be a hurdle for new sample authors but isn't it more like a know-it-once thing? Alternatives I see: 

1) Gather all files in a really shared static folder and bring them in good shape and don't hold them per package.
2) Rewrite static URLs to scope during build - more a proposal for completeness, I think there is already too much magic going on

Thanks for the other hints by the way :)

**Comment by sebastianbenz** [comment]
Why not merge all static files in a single folder during build and get rid of the static prefix?

**Pull request merged**